.github/workflows/stale.yml

```diff
@@ -26,5 +26,6 @@ jobs:
           stale-pr-message: 'This PR has been automatically marked as stale because it has not had recent activity. It will be closed in 7 days if no further activity occurs. If you wish not to mark it as stale, please leave a comment in this PR. We are sorry for this but 2 years is a long time and the code base has been changed a lot. Thanks for your contribution anyway.'
           close-pr-message: 'This PR has been automatically closed because it has not had recent activity. Sorry for that and we are looking forward to your next contribution.'
           exempt-issue-labels: 'FAQ,priority: high'
+          exempt-milestones: 'TBD'
```

Comment:
Maybe all milestoned issues and pull requests shouldn't be marked as stale? They shouldn't be closed unless they are solved.